CHANGELOG.rst

```diff
@@ -10,6 +10,7 @@ CHANGELOG
 * feature: Allow Local Serving of Models in S3
 * enhancement: Allow option for ``HyperparameterTuner`` to not include estimator metadata in job
 * bug-fix: Estimators: Join tensorboard thread after fitting
+* enhancement: Let Framework models reuse code uploaded by Framework estimators
```

Comment:
actually, sorry to nitpick - this was just sort of unfortunate timing - but we just released v1.5.0. could you make a new version entry in the changelog for this?